starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,24 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  Math.max(first, second);
 }
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
 function findLongestWord (words) {
```

Comment:
Original answer; I like it, although it is not the most performant. Try to do it in a single loop.